bootstrap/v2/server.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 OpenFaaS Authors
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+/*
```

Comment:
Is this a Golang comment for `godoc` , and if not, can it live in the readme file?